parent/pom.xml

```diff
@@ -899,6 +899,14 @@
 				<enabled>true</enabled>
 			</snapshots>
 		</repository>
+		<repository>
+			<id>redhat-maven-repository</id>
```

Comment:
use the name "jboss-ga-repository" to be aligned with naming used in official docs: https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.2/html/Development_Guide/Configure_the_JBoss_Enterprise_Application_Platform_Maven_Repository_Using_the_Maven_Settings.html